testsuite/features/support/twopence_init.rb

```diff
@@ -143,18 +140,25 @@ def get_target(host)
 # * for the usual clients, it is the full hostname, e.g. hmu-centos.tf.local
 # * for the PXE booted clients, it is derived from the branch name, the hardware type,
 #   and a fingerprint, e.g. example.Intel-Genuine-None-d6df84cca6f478cdafe824e35bbb6e3b
+# rubocop:disable Metrics/MethodLength
+# rubocop:disable Metrics/AbcSize
 def get_system_name(host)
   # If the system is not known, just return the parameter
   system_name = host
 
-  if host == 'pxeboot_minion'
+  case host
+  when 'pxeboot_minion'
     # The PXE boot minion is not directly accessible on the network,
     # therefore it is not represented by a twopence node
     output, _code = $server.run('salt-key')
     system_name = output.split.find do |word|
       word =~ /example.Intel-Genuine-None-/ || word =~ /example.pxeboot-/ || word =~ /example.Intel/ || word =~ /pxeboot-/
     end
     system_name = 'pxeboot.example.org' if system_name.nil?
+  when 'sle11sp3_terminal'
```

Comment:
Wouldn't an or statement be better here instead of multiple when? I think this won't execute anything in the first 2 when cases. You need a comma in between https://stackoverflow.com/questions/10197254/case-statement-with-multiple-values-in-each-when-block?rq=1